indico/modules/events/editing/client/js/selectors.js

```diff
@@ -29,10 +29,28 @@ function createNewCustomItemFromRevision(revision, updates) {
     createdDt: revision.createdDt,
     user: revision.submitter,
     custom: true,
+    revisionId: revision.id,
     ...updates,
   };
 }
 
+function _getLastRevertableRevisionId(revisions) {
+  let lastRevertableRevision = null;
+
+  revisions.forEach(revision => {
+    const {finalState} = revision;
+    if (finalState.name !== FinalRevisionState.none) {
+      lastRevertableRevision = revision;
+    }
+  });
+
+  // we cannot revert the initial revision
```

Comment:
you can't revert creating that revision, but you can revert e.g. accepting/rejecting it!